scylla-server-rust/src/reciever/mock_reciever.rs

```diff
@@ -0,0 +1,228 @@
+use std::time::Duration;
+
+use prisma_client_rust::{chrono, serde_json};
+use rand::Rng;
+use socketioxide::SocketIo;
+use tracing::warn;
+
+use super::ClientData;
+
+#[derive(Clone, Copy)]
+struct MockData {
+    name: &'static str,
+    unit: &'static str,
+    num_of_vals: u8,
+    min: f64,
+    max: f64,
+}
+
+impl MockData {
+    fn get_values(&self) -> Vec<String> {
+        let mut val_vec: Vec<String> = vec![];
+        // for each point, get a random number in the range
+        for _ in 0..self.num_of_vals {
+            val_vec.push(
+                rand::thread_rng()
+                    .gen_range((self.min)..(self.max))
+                    .to_string(),
+            );
+        }
+
+        val_vec
+    }
+}
+
+const BASE_MOCK_DATA: [MockData; 17] = [
```

Comment:
it'd be nice if we could house the actual data that's sent in other files, so we can separate the logic/ mock data, which would hopefully make it a little easier to add more mock data/ organize it a lil better. Not sure how annoying that is in rust tho

Reply:
@jr1221 thoughts on this?

Reply:
I'll try it don't see why it wouldn't work it'd just be a pub const.

What file name/location makes most sense?

Reply:
justthis folder, and mock_data.rs